### PR TITLE
implemented AboutTitle.tsx

### LINKED ISSUE #16: Implement AboutTitle.tsx
image

Location: **About Page**
Number of Images: **2, _about.webp, logo.webp_**

Description: Component includes one images as background. On top of the image should be the logo and a title. Title have font "font-title", and should have an underline. To overlay two tags/components, look into tailwindcss "absolute and relative".  To center the text and logo on the image, look into tailwindcss "flex, justify-center, and items-center."

Path: **ucr-hearts-website\src\components\about\AboutTitle.tsx**
[Example](https://github.com/acm-ucr/aag-website/blob/dev/src/components/Title.jsx)

### PR DESCRIPTION
**Pull request by catheiwu**
Screenshot 2024-10-15 at 1 07 14 AM

### CODE REVIEW

**Comment by catheiwu** [comment]
I was wondering how to get the required font style for the "About Us" text.

File: tailwind.config.ts [hunk 1]
```diff
@@ -9,6 +9,7 @@ module.exports = {
         brown: "#665045",
         blue: "#6D99AF",
         "light-blue": "#E8F2FB",
+        lightBrown: "#897165",
```
Comment (qhgill) [inline tailwind.config.ts]: just use the regular brown for the title and remove the extra color

**Review by qhgill — CHANGES_REQUESTED** [review]
Looks great! just use the hearts-brown color for the title.

**Comment by catheiwu** [comment]
Screenshot 2024-10-16 at 11 45 14 PM
Changed font color to just regular brown

**Review by qhgill — APPROVED** [review]
looks good!

**Pull request merged**